cleanlab/internal/util.py

```diff
@@ -291,7 +325,7 @@ def estimate_pu_f1(s, prob_s_eq_1) -> float:
     return recall**2 / (2.0 * frac_positive) if frac_positive != 0 else np.nan
 
 
-def confusion_matrix(true, pred) -> np.ndarray:
+def confusion_matrix(true: np.ndarray[LabelLike], pred: np.ndarray[LabelLike]) -> np.ndarray[Any]:
```

Comment:
Not that `LabelLike` is a concrete type, but I think it's supposed to be "vector"-like, already.

Use `LabelLike` here for now. I'll make a second pass over this at another time.